src/lib/caches/WFInfo.js

```
@@ -0,0 +1,89 @@
+        }
+      }
+      this.#cache.setKey('last_updt', Date.now());
+      this.#cache.save(true);
+
+      const end = Date.now();
+      logger.info(`WFInfo Hydration complete in ${end - start}ms`);
+    }
+  }
+
+  static {
+    this.#load();
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)